lib/davinci_dtr_test_kit/client_groups/light_ehr/dtr_light_ehr_user_response_test.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../../urls'
+require_relative '../../tags'
+
+module DaVinciDTRTestKit
+  class DTRLightEHRUserResponseTest < Inferno::Test
+    include URLs
+    id :dtr_light_ehr_user_response
+    title 'Checks for a valid user response at the supported payer endpoint'
+    description %(
+      This test verifies that a user-provided response is valid
+      when a request is made to the supported payer endpoint.
+    )
+
+    input :unique_url_id, type: :text
+
+    run do
+      load_tagged_requests(SUPPORTED_PAYER_TAG)
+      user_response = request.query_parameters['user_response']
+
+      if user_response.blank?
+        pass 'No user response provided, skipping validation and using default response.'
+      else
+        assert valid_response?(user_response), 'User response is invalid.'
+        pass 'User response is present and valid.'
+      end
```

Comment:
This isn't loading the user response when it is used. You might just always validate the request.response_body.